src/audio_call.c

```diff
@@ -919,25 +841,39 @@ void cmd_bitrate(WINDOW *window, ToxWindow *self, Tox *m, int argc, char (*argv)
 
     char *error_str;
 
-    if (argc != 1) {
-        error_str = "Must have value!";
+    Call *call = &CallControl.calls[self->num];
```

Comment:
Can you make all of these `Call *const call = &...`? That way we don't need to consider that it may be reassigned later in the function and have a different meaning there.

Reply:
Let's leave these as is.